string object_class
int32 trial
---
bool success
